File: week_06/day_3/composition/Composition_Start/app/src/main/java/example/codeclan/com/composition_start/wizard_management/MythicalBeast.java
package example.codeclan.com.composition_start.wizard_management;

/**
 * Created by user on 28/08/2017.
 */

public abstract class MythicalBeast {

    String name;

    public MythicalBeast(String name){
        this.name = name;
    }

    public String getName(){
        return this.name;
    }
}

File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management/MythicalBeast.cs
namespace WizardManagement {
   public abstract class MythicalBeast {

    private string name;

    public string Name 
    {
      get { return name; }
      set { name = value; }
    }

    public MythicalBeast(string name)
    {
      this.name = name;
    }
  }
}